### ISSUE TITLE
Review icons to include `aria-hidden`

### ISSUE THREAD

**Issue by madcampos** [techtankto/techtank-site#150]
Icons that are _purely presentational_, i.e. they don't _add_ meaning should be marked as `aria-hidden`.
This helps to avoid adding "noise" to the accessibility tree by exposing content that does not affect the comprehension of the page.

This is part of [SC 1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships) and [SC 4.1.2](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value).
